libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -150,6 +150,27 @@ const AP_Param::GroupInfo AC_AttitudeControl::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("INPUT_TC", 20, AC_AttitudeControl, _input_tc, AC_ATTITUDE_CONTROL_INPUT_TC_DEFAULT),
 
+    // @Param: LAND_R_MULT
+    // @DisplayName: Roll gain multiplier used when landed
+    // @Description: Roll gain multiplier used when landed
+    // @Range: 0 1
+    // @User: Advanced
+    AP_GROUPINFO("LAND_R_MULT", 21, AC_AttitudeControl, _land_roll_mult, 1.0),
+
+    // @Param: LAND_P_MULT
+    // @DisplayName: Pitch gain multiplier used when landed
```

Comment:
we should consider putting a Land at the front of the Display name

Reply:
Maybe 
"Landed pitch gain multiplier"
"Landed pitch gain multiplier, only active when landed" (seems to say it twice though)